lib/typescript/render/shared.ts

```diff
@@ -17,19 +17,19 @@ export interface SuggestedReplyCommand extends Command {
 export type CommandUnion = SuggestedReplyCommand;
 
 export interface MessageRenderProps {
-  message: Message;
+  message: RenderedContent;
   source: string;
   contact?: Contact;
-  lastInGroup: boolean;
+  lastInGroup?: boolean;
   invertSides?: boolean;
   commandCallback?: (command: CommandUnion) => void;
 }
 
```

Comment:
used the type _RenderedContent___ for the type that is rendered in the library and made the MessageRenderProps exclusive to Messages optional, so that it would work for both Messages & Templates